Update docs for refactored ISS integration (#21347)
home-assistant/home-assistant.github.io: source/_integrations/iss.markdown

You can check in the attributes of the sensor to see the timestamp for the next
rise of the station, its current coordinates, and the number of people in space.

{% include integrations/config_flow.md %}

<div class='note warning'>

> If you enable show on map when configurating this integration then the location attributes 

Author: ```suggestion
If you enable "Show on map" when configurating this integration then the location attributes 
```

> If you enable "Show on map" when configurating this integration then the location attributes 

Reviewer: This should not be part of the configuration, but part of an option flow, which needs to be described differently.

Author: ```suggestion
If you enable "Show on map" in the options for this integration then the location attributes 
```
